1904-11-04.xml

```diff
@@ -0,0 +1,2581 @@
+                    <p>KING'S BIRTHDAY.- The usual King's Birthday parade will be held at Mustapha
+                        next Wednesday at 10 am. THE BRINDISI MAIL (outward) will be made up at the
+                        G.P.O., Alexandria, and despatched at 8.30 am. on Monday. </p>
+                    <p>STEAMER PURCHASE- Mr. Andriadis has pur- chased the S.S. Werda from the
+                        Coastguards Administration for the sum of £700.</p>
+                    <p> QUARANTINE restrictions are now applied to arrivals in Egyptian ports from
+                        Broach, in consequence of plague existing there. </p>
+                    <p>KHEDIVIAL SOCIETY OF MEDICINE- On Saturday next the Khedivial Society of
+                        Medicine will meet at 9.30 p.m. at the Cercle Artistique. </p>
+                    <p>DRAMATIC PERFORMANCE- On Sunday next the Italian Dramatic Society of Cairo
+                        will give a performance of I Disonesti, drama in 3 acts by Rosetti. </p>
+                    <p>PARLAR DOGS- Stray, and ownerless dogs found in the Ramleh and
+                        Minet-el-Bassal districts tomorrow night will be poisoned by the police. </p>
+                    <p>THE NILE IN 1904.- The title of Sir William Willcocks's book, the publication
+                        of which we announced in yesterday's issue, is "The Nile in 1904." </p>
+                    <p>NATIONAL BANK- Mr. E. Crewe has been appointed agent, Mr. E. Wenger
+                        accountant, and Mr. A. F. Carasso cashier of the National Bank's new branch
+                        in the Mousky, Cairo, which was opened on the 1st inst. </p>
+                    <p>IRRIGATION DEPT.- Mr. Morcom, of the Irrigation Department, has been
+                        transferred from Alexandria to the 1st Circle. He will occupy the post of
+                        director of works in that Circle, in succession to Mr. Lambton. </p>
+                    <p>SOCIAL ITEMS- According to the latest statistics, it is anticipated that this
+                        winter the British colony at Cairo will be augmented by eighteen brides,
+                        including those who have already arrived since the summer. </p>
+                    <p>THE SCIPLINI CASE- Me. Fornario has presented to the clerk of the Italian
+                        Consular Court the motives of the appeal lodged against the judgment
+                        condemning M. L. Sciplini to five months' imprisonment for involuntary
+                        homicide. </p>
+                    <p>THE BIBER OUTRAGE- The director of the sagar factory at Bibeb, who was
+                        recently fired on and slightly wounded, has informed the police that be
+                        suspects one Mohamed Abdallah, who lives in an ezbeh quite close to the
+                        factory. Mohamed Abdallah has therefore been arrested. </p>
+                </div>
```

Comment:
You don't have to end the div when a new column begins. Just add the `<cb/>` between the two items.